MyCefet.Api/Services/Sigaa/InfoService.cs

```diff
@@ -41,16 +55,23 @@ public async Task<Student> GetUserInfo(string jsessionid, string username, strin
 
         private string GetInfoStudant(String jsessionid)
         {
-            var client = new RestClient("https://sig.cefetmg.br/sigaa/portais/discente/discente.jsf");
+            var client = new RestClient(SIGAA_STUDENT_PORTAL_URL);
+
+            var studentInfoHeaders = new Dictionary<string, string>
+            {
+                {ACCEPT_ENC_HEADER_KEY, ACCEPT_ENC_HEADER_VALUE },
+                {CACHE_HEADER_KEY, CACHE_HEADER_VALUE },
+                {COOKIE_HEADER_KEY, COOKIE_HEADER_VALUE + jsessionid },
+                { HOST_HEADER_KEY, HOST_HEADER_VALUE},
+                {CONTENT_HEADER_KEY, CONTENT_HEADER_VALUE },
+                {CONNECTION_HEADER_KEY, CONNECTION_HEADER_VALUE },
+                {ACCEPT_HEADER_KEY, ACCEPT_HEADER_VALUE }
+            };
```

Comment:
Before i decided to implement a "RequestSettings" generalization structure, i was using a dictionary of the request headers(keys and values). So, if you dont like the RequestSettings you can rollback to this refactoring with the Dictionary.